lib/CustomFields/components/FieldAccordion/edit-fields/shared-fields/OptionsFields/OptionsFields.js

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+import { FieldArray } from 'react-final-form-arrays';
+import { isEqual, get } from 'lodash';
+
+import {
+  MultiColumnList,
+  TextField,
+  RadioButton,
+  IconButton,
+  Col,
+  Label,
+  ConfirmationModal,
+} from '@folio/stripes-components';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+
+import css from './OptionsFields.css';
+import { AddOptionButton } from '../../components';
+
+const propTypes = {
+  fieldNamePrefix: PropTypes.string.isRequired,
+  label: PropTypes.string,
```

Comment:
I'd rather name this prop "customFieldLabel". Every option has its label too, so it's a bit confusing